docs/backup-restore.md

```diff
@@ -3,30 +3,63 @@
 Since Scalar DB provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
 This document sets out some guidelines for backing up and restoring the databases that Scalar DB supports.
 
-## Cassandra
+## Scalar DB server
+
+Since Scalar DB server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, you can easily pause the server to stop accepting incoming requests and drain in-flight requests.
+You can use paused duration and choose one of the following ways depending on the underlining database to make your databases transactionally-consistent even after restored.
```

Comment:
This explanation should come to the next section?

Reply:
Sorry for the inconvenience.
Document updated with a new structure.